include/git2/diff.h

```diff
@@ -479,14 +487,15 @@ typedef int(*git_diff_binary_cb)(
 /**
  * Structure describing a hunk of a diff.
  */
-typedef struct {
-	int    old_start;     /**< Starting line number in old_file */
-	int    old_lines;     /**< Number of lines in old_file */
-	int    new_start;     /**< Starting line number in new_file */
-	int    new_lines;     /**< Number of lines in new_file */
-	size_t header_len;    /**< Number of bytes in header text */
-	char   header[128];   /**< Header text, NUL-byte terminated */
-} git_diff_hunk;
+typedef struct git_diff_hunk git_diff_hunk;
+struct git_diff_hunk {
```

Comment:
Any particular reason to split off the struct and type definitions? IIRC the doc parser does not like this (and also me stylistically)